src/main/java/frc/robot/utils/DeltaAngleSpeedCalcUtil.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.utils;
+
+import edu.wpi.first.math.Pair;
+import edu.wpi.first.math.util.Units;
+import frc.robot.Constants;
+
+public class DeltaAngleSpeedCalcUtil {
+  private static final double SPEAKER_HEIGHT_INCHES = Units.inchesToMeters(77.9528);
+  private static final double SHOOTER_SPEED_MPS = Constants.PLACEHOLDER_DOUBLE;
+
+  /**
+   *
+   * @param tangentialVelocityMPS: see google doc
+   * @param radialVelocityMPS: see google doc
+   * @param distFromSpeakerMeters: distance from speaker :)
+   * @param currentAzimuthAngleDegrees: see google doc
+   * @return: delta of azimuth and elevation angle degrees (change in angle)
+   */
+  public static Pair<Double, Double> calcDeltaAngle(
+      double tangentialVelocityMPS,
+      double radialVelocityMPS,
+      double distFromSpeakerMeters,
+      double currentAzimuthAngleDegrees) {
+
+
+    /** 
+     * :)))))))))))))))))))
+     * https://docs.google.com/document/d/1rIMmzDA1EMH7NQexIMrNYAD-_KFLe9cL4zPLXdl0lek/edit?usp=sharing
+     */
+    double calculatedAzimuthAngleDegrees = Math.atan(tangentialVelocityMPS / SHOOTER_SPEED_MPS);
+    double deltaAzimuthAngleDegrees = currentAzimuthAngleDegrees - calculatedAzimuthAngleDegrees;
```

Comment:
Isn't `calculatedAzimuthAngleDegrees` itself the delta?

Reply:
Actually I think you're right.. good point.